apps/website/public/.well-known/security.txt

```diff
@@ -1,4 +1,4 @@
 Contact: mailto:opensource@alveussanctuary.org
-Policy: https://github.com/alveusgg/alveusgg/blob/main/SECURITY.md
-Expires: 2025-01-31T04:00:00.000Z
+Policy: https://github.com/alveusgg/alveusgg/security/policy
+Expires: 2026-01-31T04:00:00.000Z
```

Comment:
```suggestion
```
👀 I wonder if we should just remove the expires line? This info should be timeless now?

Reply:
Yeh, let's remove it, ty